weaviate/collection/grpc.py

```diff
@@ -1161,20 +1171,20 @@ def get_flat(
 
     def get_options(
         self,
-        returns: ReturnValues,
+        returns: ReturnValues[Model],
```

Comment:
...and this for the ORM?

Reply:
Specifying the Generic here doesn't affect its use of the UX, it's just required for static type checking